clients/src/test/java/org/apache/kafka/common/network/SelectorTest.java

```diff
@@ -83,7 +83,7 @@ public void setUp() throws Exception {
         this.channelBuilder = new PlaintextChannelBuilder();
         this.channelBuilder.configure(configs);
         this.metrics = new Metrics();
-        this.selector = new Selector(5000, this.metrics, time, "MetricGroup", channelBuilder, new LogContext());
+        this.selector = new Selector(5000, 100, this.metrics, time, "MetricGroup", channelBuilder, new LogContext());
```

Comment:
Not sure why this was changed. This is the client-side selector. We are not delaying on clients? (Multiple occurrences)